hummingbot/connector/derivative/bybit_perpetual/bybit_perpetual_websocket_adaptor.py

```diff
@@ -50,6 +78,24 @@ async def subscribe_to_instruments_info(self, symbols: Optional[List[str]] = Non
                                 self._payload_field_name: [f"{CONSTANTS.WS_INSTRUMENTS_INFO_TOPIC}.{symbol_filter}"]}
         await self.send_request(subscription_message)
 
+    async def subscribe_to_positions(self):
+        subscription_message = {self._operation_field_name: self._subscription_operation,
+                                self._payload_field_name: [f"{CONSTANTS.WS_SUBSCRIPTION_POSITIONS_ENDPOINT_NAME}"]}
+        await self.send_request(subscription_message)
+
+    async def subscribe_to_orders(self):
+        subscription_message = {self._operation_field_name: self._subscription_operation,
+                                self._payload_field_name: [f"{CONSTANTS.WS_SUBSCRIPTION_ORDERS_ENDPOINT_NAME}"]}
+        await self.send_request(subscription_message)
+
+    async def subscribe_to_stop_orders(self):
+        subscription_message = {self._operation_field_name: self._subscription_operation,
+                                self._payload_field_name: [f"{CONSTANTS.WS_SUBSCRIPTION_STOP_ORDERS_ENDPOINT_NAME}"]}
+        await self.send_request(subscription_message)
+
+    async def receive_str(self, *args, **kwars):
```

Comment:
If all the messages we receive are in JSON format, we should stick to the `receive_json` function instead of using the `receive_str` and then parse it using the JSON module